sessions/functionals.qmd

```diff
@@ -593,6 +593,8 @@ user_info_df %>%
   select(where(is.character))
 ```
 
+What is this trim_filepath_for_book?
```

Comment:
Yea, I forgot to make it not show up in the website. Its for making the column match what they will see.